src/api/search/jest.config.js

```diff
@@ -0,0 +1,8 @@
+const baseConfig = require('../../../jest.config.base');
+
+module.exports = {
+  ...baseConfig,
+  rootDir: '../../..',
```

Comment:
I might be wrong, but I think you have to specify the `jest.setup.js` file here, like this: https://github.com/Seneca-CDOT/telescope/blob/19e4dfc9587d17b4948a7229ff4f5b6d5c0f02e6/test/jest.config.js#L5

We need to do the same for the posts.  If this is working already, I'm not clear why it works without setting this.  The [docs](https://jestjs.io/docs/configuration#setupfiles-array) don't lead me to believe this would happen automatically.

Reply:
Might be because we don't have Elasticsearch mocks in the tests yet to see if its setup correctly.

Reply:
But why do the Posts tests work, which also mock redis this way, but don't include the setup script in the config?  Seems like a bug to me.

Reply:
So I checked all of the other config test. All of them did not have to specify the `jest.setup.js`

Reply:
Right, we only need it very rarely.  I'm saying that I think we are doing it wrong everywhere else.

Reply:
I think somehow `MOCK_REDIS` and `MOCK_ELASTIC` are already set to `'1'`
I printed out `process.env` on the master branch locally (without the tests for search)
![image](https://user-images.githubusercontent.com/32626950/151438916-2737feb4-2cea-4c06-a875-49ba24d664b6.png)

Reply:
You're right, it's happening here https://github.com/Seneca-CDOT/telescope/blob/master/jest.config.base.js#L4.

I think it's a good idea to not depend on this, so we should be adding the `setupFiles` key to your search test config.  In another PR, we should do the same in the posts tests.  Can one of you do that too?

Reply:
Yup, I'll do it for Posts

Reply:
@rclee91 if I understood correctly for this PR we need to include `setupFiles: ['<rootDir>/src/api/search/jest.setup.js']`?

Reply:
Yup, looks like it

Reply:
We should get this finished, as it blocks all other test work from landing.  @JiaHua-Zou can you wrap it up?

Reply:
sorry didn't see this message. I just push a new PR